site/src/components/DropdownMenu/DropdownMenu.tsx

```diff
@@ -0,0 +1,198 @@
+import * as DropdownMenuPrimitive from "@radix-ui/react-dropdown-menu";
+import { Check, ChevronRight, Circle } from "lucide-react";
+import * as React from "react";
```

Comment:
I have been following the pattern of not using the * import for React for the other shadcn/ui components as we don't generally have this pattern in Coder